testsuite/policy/authorization/auth_config.py

```diff
@@ -84,6 +84,16 @@ def wait_for_ready(self):
             assert success, f"{self.kind()} did not get ready in time"
             self.refresh()
 
+    def wait_for_accepted(self):
+        """Waits until authorization object reports accepted status"""
+        with oc.timeout(90):
+            success, _, _ = self.self_selector().until_all(
```

Comment:
How about utilizing `wait_until`?

Reply:
I just copy/pasted `wait_for_ready` function that is defined just above this one. I will take a look into refactoring using `wait_until` and do it if there are no catches there.

Reply:
I refactored both `wait_for_accepted` and `wait_for_ready` using `wait_until` under the hood. It seems to work however, the original implementation also called `self.refresh()` at the very end which is not the case anymore. Is that `self.refresh()` call really needed?
Furthermore, I added `any(x.type == "Enforced" for x in obj.model.status.conditions)` into `wait_for_ready` because my understanding is that it should only return `True` if the policy is Enforced successfully (that's when policy is "ready" if I got it right).

Reply:
This will be addressed by https://github.com/Kuadrant/testsuite/pull/443 I will rebase once it gets merged.